### PR TITLE
missing gradle dir in start 

### PR DESCRIPTION
**Pull request by DannyS03**
https://github.com/OpenLiberty/guide-gradle-intro/issues/8
https://github.com/OpenLiberty/guide-gradle-intro/pull/55#pullrequestreview-321134616

### CODE REVIEW

**Comment by gkwan-ibm** [comment]
@DannyS03 

Need to add the bold statement at the **Testing Gradle** section in between following lines:

If you do not have Gradle installed...

**Make sure that the JAVA_HOME environment variable is set or `java` application can be run.**

Test that Gradle is installed properly by running the following command:

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt would you review this PR?

File: README.adoc [hunk 1]
```diff
@@ -79,8 +79,7 @@ is in the `start/src` directory:
 
 == Testing Gradle
 
-If you do not have Gradle installed, running the Gradle Wrapper will automatically install Gradle. To learn 
-more about Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle documentation^].
+If you do not have Gradle installed, make sure that the `JAVA_HOME` environment variable is set, or that the java application can run. Running the Gradle Wrapper will automatically install Gradle. To learn more about Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle Wrapper documentation^].
```
Comment (Charlotte-Holt) [inline README.adoc]: Suggested update:
"If you do not have Gradle installed, make sure that the `JAVA_HOME` environment variable is set, or that the **Java** application can run. Running the Gradle Wrapper **automatically installs** Gradle. To learn more about **the** Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle Wrapper documentation^]."

**Comment by gkwan-ibm** [comment]
@DannyS03 would you fix what Charlotte suggested?

File: README.adoc (hunk 1 above)
Comment (DannyS03) [reply README.adoc]: Thanks!

**Pull request merged**